src/App/Fossa/FossaAPIV1.hs

```diff
@@ -457,15 +458,17 @@ getIssues apiOpts ProjectRevision{..} = fossaReq $ do
 
 ---------------
 
-attributionEndpoint :: Url 'Https -> Int -> Locator -> Url 'Https
-attributionEndpoint baseurl orgId locator = baseurl /: "api" /: "revisions" /: renderLocatorUrl orgId locator /: "attribution" /: "json"
+attributionEndpoint :: Url 'Https -> Int -> Locator -> ReportOutputFormat -> Url 'Https
+attributionEndpoint baseurl orgId locator ReportJson = baseurl /: "api" /: "revisions" /: renderLocatorUrl orgId locator /: "attribution" /: "json"
+attributionEndpoint baseurl orgId locator ReportSpdx = baseurl /: "api" /: "revisions" /: renderLocatorUrl orgId locator /: "attribution" /: "full" /: "spdx"
+attributionEndpoint baseurl orgId locator ReportMarkdown = baseurl /: "api" /: "revisions" /: renderLocatorUrl orgId locator /: "attribution" /: "full" /: "MD"
 
-getAttribution ::
+getAttributionJson ::
```

Comment:
Why do we need to handle JSON in a special way?  It looks like this function is only used within this module (I only see four uses of this word in the diff).  Maybe we should just get rid of it and just use bytestring output.

Reply:
The querystring options passed to the server are different.  I think we're not sure if we can use them everywhere else yet.

Reply:
Yep, the query string is unused for the backend route used by the other formats. While technically it wouldn't break anything to pass them anyway today, I don't consider it correct.